specs/src/specs/data_structures.md

```diff
@@ -498,16 +498,20 @@ For shares **with a reserved namespace ID through [`NAMESPACE_ID_MAX_RESERVED`](
 > **Note** The first [`NAMESPACE_ID_BYTES`](./consensus.md#constants) of a share's raw data `rawData` is the namespace ID of that share, `namespaceID`. The next [`SHARE_INFO_BYTES`](./consensus.md#constants) bytes are for share information.
 
 - If this is the first share of a sequence, the next 1 to 10 bytes contain a [varint](https://developers.google.com/protocol-buffers/docs/encoding) of the `uint64` length of the sequence that follows, in bytes.
-- The next [`SHARE_RESERVED_BYTES`](./consensus.md#constants) bytes is the starting byte of the length of the [canonically serialized](#serialization) first request that starts in the share, or `0` if there is none, as a one-byte big-endian unsigned integer (i.e. canonical serialization is not used). In the example below, with a share size of `256` the reserved byte would be `80` (or `0x50` in hex).
+- The next [`SHARE_RESERVED_BYTES`](./consensus.md#constants) bytes are the starting byte of the length of the [canonically serialized](#serialization) first request that starts in the share, or `0` if there is none, as an unsigned [varint](https://developers.google.com/protocol-buffers/docs/encoding).
 - The remaining [`SHARE_SIZE`](./consensus.md#constants)`-`[`NAMESPACE_ID_BYTES`](./consensus.md#constants)`-`[`SHARE_INFO_BYTES`](./consensus.md#constants) `-` 1 to 10 bytes (if this is the first share of a sequence) `-` [`SHARE_RESERVED_BYTES`](./consensus.md#constants) bytes are transactions, intermediate state roots, or evidence data depending on the namespace of ths share. Each transaction, intermediate state root, or evidence is prefixed with a [varint](https://developers.google.com/protocol-buffers/docs/encoding) of the length of that unit.
 - If there is insufficient transaction, intermediate state root, or evidence data to fill the share, the remaining bytes are filled with `0`.
 
-First share in a sequence:
+First share in a sequence:<br>
```

Comment:
[Non Blocking]
Doesn't it make sense not to use `<br>` as throughout the repo we're opting for new line?
I'm fine with switching to `<br>` notation

Reply:
the `<br>` is used to add an explicit newline between this sentence and the image. Without `<br>` the image may render on the same line as the sentence.

<img width="1101" alt="Screen Shot 2022-11-01 at 3 18 21 PM" src="https://user-images.githubusercontent.com/3699047/199255116-07eeebb5-faa1-49d8-834f-bb333e59263e.png">

Reply:
Aaah got you, thanks :+1:

Reply:
Can't you just add a newline?

Reply:
Yea, not sure why that didn't occur to me. Will fix in an upcoming commit.

Reply:
Resolved by https://github.com/celestiaorg/celestia-app/pull/939/commits/ea4cea5ba03dfc27e0e74c33675b0725832a20b5